test/files/jvm/string-switch/Test.scala

```diff
@@ -0,0 +1,16 @@
+import scala.tools.partest.BytecodeTest
+import scala.tools.asm
+import scala.collection.JavaConverters._
+import scala.PartialFunction.cond
+
+object Test extends BytecodeTest {
```

Comment:
Instead of using partest, adding this test to https://github.com/scala/scala/blob/2.13.x/test/junit/scala/tools/nsc/transform/patmat/PatmatBytecodeTest.scala might be preferrable.

Reply:
If you'd prefer it, then I prefer it. Everyone uses JUnit or scalatest (and soon, scala-verify) but we're the lucky few who get to use partest.